README.md

```diff
@@ -89,61 +89,143 @@ builds like `gcc` generate a *lot* of output), and builds timing out
+
+# Clone the litex-conda-compilers repository
 git clone https://github.com/litex-hub/litex-conda-compilers.git
 cd litex-conda-compilers
-./.travis/conda-get.sh
-
-# Adapted from .travis/common.sh
-get_built_package() {
-   ./.travis/conda-env.sh render --output "$@" 2>/dev/null | grep conda-bld | grep tar.bz2 | tail -n 1 | sed -e's/-[0-9]\+\.tar/*.tar/' -e's/-git//'
-}
-
-# Anchor the build date/time, so we have predictable versions and filenames
-# amongst related packages, and to make it easy to do package installs.
-#
-# Either to current date/time at the start of the build:
-#
-# export DATE_NUM="$(date -u +%Y%m%d%H%M%S)"
-# export DATE_STR="$(date -u +%Y%m%d_%H%M%S)"
-#
-# Or lock to date/time of the last commit on git, as Travis CI config does
-# (see .travis/common.sh)
-#
-export DATE_TS="$(git log --format=%ct -n1)"
-export DATE_NUM="$(date --date=@${DATE_TS} -u +%Y%m%d%H%M%S)"
-export DATE_STR="$(date --date=@${DATE_TS} -u +%Y%m%d_%H%M%S)"
-
-# Combinations taken from .travis.yml
-TOOLCHAIN_ARCH=lm32
-export PACKAGE TOOLCHAIN_ARCH
-
-cd conda-hdmi2usb-packages
-
-for PACKAGE in binutils gcc/nostdc gcc/newlib; do
-  ./.travis/conda-env.sh build --check "${PACKAGE}"   # Downloads and caches stuff
-  ./.travis/conda-env.sh build         "${PACKAGE}"   # Actually build package
-  CONDA_OUT="$(get_built_package ${PACKAGE})" # Calculate output package
-  ./.travis/conda-env.sh install       "${CONDA_OUT}"
-done
 ```
 
+### Required environment variables
+
+The recipe might require exporting some additional environment variables.
+Such variables must be set before preparing the recipe, which is done
+with one of the commands from the next subsection.
+
+Currently required additional environment variables are:
+* `TOOLCHAIN_ARCH` (by: `binutils`, `gcc/*`, `gdb` and `toolchain/linux-musl`
+recipes) – must contain a supported architecture name, ie. one of these:
+  * `lm32` (excluding `gcc/linux-musl` and `toolchain/linux-musl` recipes),
+  * `or1k`,
+  * `ppc64le` (only `binutils` and `gcc/linux-musl` recipes),
+  * `riscv32`,
+  * `riscv64`,
+  * `sh`.
+
+The optional `DATE_NUM` and `DATE_STR` environment variables are used by the
+recipes to set the `build/number` and `build/string` keys during preparation.
+In case they're unset, recipe preparation provides them with default values
+based on the latest commit's committer date from the repository holding the
+recipe.
+
+### Preparing and building the package
+
+After getting all prerequisites and setting the required variables, it
+is recommended to prepare the recipe before building, as it gives you
+the advantages described on [the `conda-build-prepare`'s
+GitHub page](https://github.com/litex-hub/conda-build-prepare).
+The tool is also used within the CI, which makes the locally built
+package much more similar to the one built by the CI workflow.
+
+The tool is a Python module.
+The only required arguments are the path to the recipe corresponding to the
```

Comment:
Maybe out of scope of this PR, but it'd be interesting to generate this details as a canonical help (docstrings) in Python. Then, copy the output of `-h` here.

Paebbels/pyAttributes might be useful.

Reply:
I don't think that users should really be informed about all the details of `conda-build-prepare`. If the primary goal is to make building process similar to the CI one then it's best when the same arguments as in that CI are used.

Didn't know the pyAttributes though, I'll check it out.

Reply:
You might've been confused about what tool this was as there was "the tool". I made the whole paragraph much more precise now.

Reply:
My point is, are users expected to execute `conda-build-prepare` or is it for developers only? If it's going to be executed by users, I think that a list output is desirable:

- `--dir`: ...
- `--channels`: ...
- `--packages`: ...

By using pyAttributes or a similar solution, the amount of info shown to users can be decided.

Reply:
Anybody who wants to build a package as it's done in the CI should use *conda-build-prepare* and with the `--channels` and `--packages` the same as in README (=CI).

I've changed the whole section on *conda-build-prepare* arguments and I think it's much more comprehensible now. I've explained thoroughly the options that user might be interested in, what should be passed to the remaining ones and why.